### PR TITLE
DOC-12761: Add new features from 3.2.1 to what's new page

### PR DESCRIPTION
**Pull request by ElliotFrancisHunter**
PR to add features introduced in 3.2.1 to the what's new page. Also linked out to the source documentation material per platform

### CODE REVIEW

**Comment by ElliotFrancisHunter** [comment]
Build failure has already been addressed in another PR and is unrelated to docs.

File: modules/ROOT/pages/cbl-whatsnew.adoc [hunk 1]
```diff
@@ -14,9 +14,49 @@ include::partial$_show_page_header_block.adoc[]
...
+If you're upgrading from 2.x, refer to the appropriate upgrade page -- see: <<lbl-upgrade>>
 Users should be able to upgrade to 3.1.x from 3.0.x without manual intervention.
 
+== Release 3.2.1 (November 2024)
+
+=== New Features
+
+==== Array UNNEST and the Array Index
+
+You can use UNNEST in queries to unpack arrays within a document into individual rows. This capability makes it possible to join them with its parent object in the query.
```
Comment (simon-dew) [inline modules/ROOT/pages/cbl-whatsnew.adoc]: Number agreement

```suggestion
You can use UNNEST in queries to unpack arrays within a document into individual rows. This capability makes it possible to join them with their parent object in the query.
```

File: modules/ROOT/pages/cbl-whatsnew.adoc [hunk 2]
```diff
@@ -14,9 +14,49 @@ include::partial$_show_page_header_block.adoc[]
...
+
+UNNEST is used within the FROM clause and can be chained to perform multi-level UNNEST.
```
Comment (simon-dew) [inline modules/ROOT/pages/cbl-whatsnew.adoc]: These (FROM clause, chaining UNNESTs) are separate but related ideas. I think I'd prefer them to be two separate sentences. I'm not going to insist on it though.

**Review by simon-dew — DISMISSED** [review]
Just one minor correction and one comment

File: modules/ROOT/pages/cbl-whatsnew.adoc (hunk 2 above)
Comment (ElliotFrancisHunter) [reply modules/ROOT/pages/cbl-whatsnew.adoc]: Done

**Review by simon-dew — APPROVED** [review]
LGTM 👍

**Pull request merged**